lib/BC/SleighLifter.cpp

```diff
@@ -313,6 +325,19 @@ class SleighLifter::PcodeToLLVMEmitIntoBlock : public PcodeEmit {
   }
 
  public:
+  llvm::BasicBlock *GetOrCreateBlock(size_t target) {
+    auto blk = block_start_index_to_basic_block.find(target);
+    if (blk != block_start_index_to_basic_block.end()) {
+      return blk->second;
+    }
```

Comment:
^ is a separate function. Then here
```
if ( auto blk = fetch( target ) )
  return blk;
... create it ...
```

Reply:
[afa64c3](https://github.com/lifting-bits/remill/pull/645/commits/afa64c318388068d6459e908650c178bf5dfd2e1)